backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/dsl/impl/ElasticsearchInstantIndexFieldTypeContext.java

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
Not related to this commit, but just so I remember to address the issue: we need to update the documentation.
I will do that once the rest of the comments are addressed.